dependency.lic

```diff
@@ -49,10 +49,26 @@ class ArgParser
   end
 
   def display_args(data)
-    data.each do |def_set|
-      respond ''
-      respond "  ;#{Script.current.name} " + def_set.map { |x| format_item(x) }.join(' ')
-      def_set.each { |x| respond "   #{(x[:display] || x[:name]).ljust(12)} #{x[:description]} #{x[:options] ? '[' + x[:options].join(', ') + ']' : ''}"; }
+    # Display help output for script arguments
+    unless Script.current.name == "bootstrap"
+      data.each do |def_set|
+        respond ''
+        def_set.each { |x| respond " SCRIPT SUMMARY: #{x[:description]} " if x[:name].to_s == "script_summary"; }
```

Comment:
```suggestion
        def_set
        .select { |x| x[:name].to_s == "script_summary" }
        .each { |x| respond " SCRIPT SUMMARY: #{x[:description]} " }
```

Reply:
Thanks for the suggestion. I still struggle through these and end up having to do quite a bit of live testing to get anything right. Let me digest this one.